RNSound/RNSound.m

```diff
@@ -47,6 +47,9 @@ -(void) audioPlayerDidFinishPlaying:(AVAudioPlayer*)player
   if (key == nil) return;
 
   @synchronized(key) {
+    if (!flag) {
+      [self setOnPlay:NO forKey:key];
```

Comment:
I think that `[self setOnPlay:NO forKey:key]` should be called regardless of `flag` because the player has stopped regardless of `flag`. Please see `flag` parameter description in the following link https://developer.apple.com/documentation/avfoundation/avaudioplayerdelegate/1389160-audioplayerdidfinishplaying

Reply:
Yes! I definitely agree with you.